tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "baseUrl": ".",
     "paths": {
       "protractor": ["test/protractor.d.ts"]
-    }
+    },
+    "esModuleInterop": true
```

Comment:
@zhukovka 
But also don't forget to remove this 😁 
Thanks 🙏

Reply:
done